src/python/bot/fuzzers/engine.py

```diff
@@ -72,3 +78,20 @@ def cleanse(self, target_path, input_path, output_path, max_time):
     """Optional: scrub a testcase of potentially sensitive bytes. Returns a
     bool."""
     raise NotImplementedError
+
+
+def register_engine(name, impl):
```

Comment:
Can we use `impl.name` instead of having name as a param?

Reply:
Done.